src/main/java/com/minenash/pocketwatch/Pocketwatch.java

```diff
@@ -28,7 +28,7 @@ public class Pocketwatch implements ClientModInitializer {
 	@Override
 	public void onInitializeClient() {
 
-		CONFIG.whitelist().replaceAll(id -> Identifier.of(id).toString());
+		CONFIG.whitelist().removeAll(Arrays.asList("", null)).replaceAll(id -> new Identifier(id).toString());
```

Comment:
Does this change compile on 1.21? I don't think that constructor for Identifier exists anymore.

Reply:
Oh, you're probably right. Let me fix that.